src/main/java/com/deavensoft/paketomat/courier/CourierServiceImpl.java

```diff
@@ -121,16 +126,30 @@ public boolean isCityInList(String city, List<City> citiesToDispatch) {
     public List<Package> deliverPackageInPaketomat(List<Package> packages) {
         for (Package p : packages) {
             centerService.updateStatus(p.getCode(), Status.IN_PAKETOMAT);
-            sendMailToUser(p.getUser().getEmail());
+            if (p.getPaid() == Paid.PAID)
+                sendMailToUser(p.getUser().getEmail(), Paid.PAID);
+            else if (p.getPaid() == Paid.NOT_PAID) {
+                sendMailToUser(p.getUser().getEmail(), Paid.NOT_PAID);
+            } else if (p.getPaid() == Paid.UN_SUCESSFULL) {
+                sendMailToUser(p.getUser().getEmail(), Paid.UN_SUCESSFULL);
+
+            }
```

Comment:
- [ ] Extract in a method

Reply:
Done.

Reply:
Well done

Reply:
But maybe the name like _sendEmailToUser_ better describes the method intention.

Reply:
?

Reply:
Done